.pre-commit-config.yaml

```diff
@@ -1,4 +1,20 @@
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: check-symlinks
+      - id: check-toml
+      - id: check-vcs-permalinks
+      - id: detect-private-key
```

Comment:
`detect-private-key` is a really good idea. Wondering if there's something similar / we could write something for API keys?

Reply:
Good idea!
For this PR, this is probably out of scope, so I opened a new issue for this:
- https://github.com/digitalfabrik/integreat-cms/issues/2250